public/content/documentation/web/component/heading.md

```diff
@@ -6,75 +6,179 @@ How to test a heading: h1, h2, h3, h4, h5, h6
 
-How to test a header
+How to test a heading
 
 1. Test keyboard only, then screen reader + keyboard actions
 
-   - Skip-links: Focus moves directly to the header or navigation
-
-   - Tab: Nothing, headings are not focusable unless they are actionable
-
-   - Arrow-keys: Headings are browsed
+      - Tab: Nothing, headings must not be focusable
+      - Arrow-keys: Browses headings (when using screen reader)
 
 2. Test mobile screenreader gestures
 
-   - Swipe: Focus moves directly to the header or navigation
-
-   - Doubletap: This typically activates most elements
+      - Swipe: The screenreader reads the heading and its level
 
 3. Listen to screenreader output on all devices
 
-   - It is discoverable with screenreader shortcuts as header/banner landmark
-
-   - Group: It typically contains the name and primary navigation of the website
+      - Name: The heading's purpose and level must be clear
+      - Role: It identifies itself as a heading and its level
+      - Group: It is logically ordered, starting with a single h1, sections titled by h2, and subsections with h3
 
 Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/heading](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/heading)
 
 ## Gherkin
 
 ### #a11y - Web Accessibility Acceptance Criteria
 
-How to test a header
+How to test a heading
 
-GIVEN THAT I am on a page with a header landmark
+GIVEN THAT I am on a page with a heading
 
 1. Keyboard for mobile & desktop
 
-   - WHEN I use the tab key to enter the web browser window I SEE focus is strongly visually indicated on interactive components
+      - WHEN I use the arrow key to browse to a heading I SEE the heading comes into view
+      - WHEN when I use the tab key I SEE nothing happens to the heading because headings must NOT be focusable
 
 2. Desktop screenreader
 
-   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND
+      - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND 
+      - I use the arrow key to browse to a heading
+         - I HEAR The heading's purpose and level must be clear
+         - I HEAR It identifies itself as a heading and its level
+         - I HEAR It is logically ordered, starting with a single h1, sections titled by h2, and subsections with h3
+      - WHEN when I use the tab key I HEAR nothing happens to the heading because headings must NOT be focusable
 
-   - I use the tab key to enter the web browser window
+3. Mobile screenreader
 
-   - I HEAR It is discoverable with screenreader shortcuts as header/banner landmark
+      - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
+      - I swipe to browse to a heading
+         - I HEAR The heading's purpose and level must be clear
+         - I HEAR It identifies itself as a heading and its level
+         - I HEAR It is logically ordered, starting with a single h1, sections titled by h2, and subsections with h3
```

Comment:
I know this is nitpicky, I've been getting comments on it too :), but we've been changing the "It" to "it". Just make sure the letters following "HEAR" are lowercase. Same for #2 above.